app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -22,6 +22,19 @@ def button
     @refresh_div = "main_div" # Default div for button.rjs to refresh
     tag(self.class.model) if params[:pressed] == "#{params[:controller]}_tag"
     return if ["#{params[:controller]}_tag"].include?(params[:pressed]) && @flash_array.nil? # Tag screen showing
+
+    # Handle scan
+    if params[:pressed] == "container_image_scan"
+      scanimages
```

Comment:
@oschreib I think `scan_images` was suggested here (both from @abonas and @blomquisg )

I also prefer it so we'll start writing code as it should be (instead of changing it later).